app/controllers/account/users_controller.rb

```diff
@@ -18,6 +18,22 @@ def update
     end
   end
 
+  def new_assign_osbb
+    @user = authorize current_user
+  end
+
+  def assign_osbb # rubocop:disable Metrics/AbcSize
+    osbb = Osbb.find_by(name: user_osbb_params[:osbb_id])
+    if current_user.update(osbb_id: osbb.id, role: :members)
+      SendNotificationAboutNewMemberWorker.perform_async(osbb.id, current_user.id)
```

Comment:
make name shorter NewMemberWorker should be enough
it's too complicate and redundant to have such long naming